### PR TITLE
Revert "Hugboxes heresy without adding weird metagame (#519)"

### PR DESCRIPTION
**Pull request by destrucktoid**
This reverts commit 3fd5db0532fc390b9d8be8e83f162444b23bbd5d.

## About The Pull Request

Revert PR for #519. 

## Why It's Good For The Game

The original PR to change heretic sacrificing was generally given negative feedback and yet was merged anyway for some reason? Regardless of that it's a bad change nonetheless. The heretic brain trauma WAS revertible via usage of a regen extract from xenobio, making the heretic brain trauma now super easy to remove is kind of stupid. 

"Ah yes, i've been sent to a realm of gods beyond mortal comprehension and my brain was scarred forever as a result... Or as it turns out just till i get a bit of minor brain surgery, not even a lobotomy."

How about instead of complaining about the heretic brain trauma, instead you head over to xenobio and get a regen extract. 

This change was not necessary and is massive overkill as well as being massively over dramatized in how bad it is. Your standard spessman can be sacrificed and go back to do their job with no issues, the only time this was ever and issue is if you was security and honestly, if you die/lose to an antag you shouldn't be rushing back to go kill them again anyway, we specifically added a rule that lets antags RR security if they do this.

TL:DR Original PR was a bad change that shouldn't have been merged, it cited the fact the trauma was irreversible as it's main reason to be merged and that was just plainly untrue as xenobio can in fact heal the trauma.

## Changelog

🆑 
qol: Reverts Heretics no longer giving surgery resistant traumas
🆑

### CODE REVIEW

**Comment by rintherat** [comment]
Uh... there is a rule around RRing Security if they rush straight back to the fight tho. You CAN RR Security for doing that.

**Comment by Majkl-J** [comment]
the overwhelmingly negative feedback was you and one other guy complaining about how a very specific niche mechanic fixes it because ????

**Comment by yourdoom9898** [comment]
> This change was not necessary and is massive overkill as well as being massively over dramatized in how bad it is. Your standard spessman can be sacrificed and go back to do their job with no issues

Said by someone who never actually had to deal with the supernatural trauma and the fact that you'll now get hardstunned for the rest of the shift repeatedly unless you turn off your radio and hide in a corner where no one will interact with you so if someone says the word "wizard" or "magic" you don't freak the fuck out

**Comment by Serxule** [comment]
yeaaaaah the supernatural phobia is insanely bad, anything remotely related to "supernatural" (stuss like the chaplains jumpsuit counts)  will cause the phobia to trigger, of which most things make the game literally unplayable, ranging from constant stuns to blindness

**Comment by destrucktoid** [comment]
> Uh... there is a rule around RRing Security if they rush straight back to the fight tho. You CAN RR Security for doing that.

Yes, i know, that's why i said this: "we specifically added a rule that lets antags RR security if they do this." in my original post. My point around security having to actually go to xenobio and get a extract rather than just instantly rushing to go kill the heretic

**Comment by destrucktoid** [comment]
> the overwhelmingly negative feedback was you and one other guy complaining about how a very specific niche mechanic fixes it because ????

I never actually posted in your PR. I didn't even notice it or i would've posted complaints about it then and tried to push for it to not be merged in the first place.

**Comment by destrucktoid** [comment]
> > This change was not necessary and is massive overkill as well as being massively over dramatized in how bad it is. Your standard spessman can be sacrificed and go back to do their job with no issues
> 
> Said by someone who never actually had to deal with the supernatural trauma and the fact that you'll now get hardstunned for the rest of the shift repeatedly unless you turn off your radio and hide in a corner where no one will interact with you so if someone says the word "wizard" or "magic" you don't freak the fuck out

The word magic and wizard both don't trigger the trauma actually. Also i HAVE dealt with the trauma, there was a HoS shift i had where i got sacrificed by one of the 5 heretics we had that shift. You want to know how i dealt with it? I got a regen extract from xenobio and that fixed my trauma. I then proceeded to kill/arrest 3 of the heretics.

**Comment by destrucktoid** [comment]
> yeaaaaah the supernatural phobia is insanely bad, anything remotely related to "supernatural" (stuff like the chaplains jumpsuit counts) will cause the phobia to trigger, of which most things make the game literally unplayable, ranging from constant stuns to blindness

It's really not as bad as people make it out to be. Yes the chaplain's jumpsuit triggers it as well as other "supernatural" stuff but in reality how often as a botanist or a chemist are you going to see a chaplain's jumpsuit? Or hear the word "heretic" (considering with metaprotections most people aren't allowed to call them heretics). I do want to repeat myself once more however since people don't seem to get the hint. _The trauma always **was** fixable, you just need a regenerative extract from xenobio!_

**Comment by Serxule** [comment]
still, majority of people arent going to know that xenobio fixes anything

**Comment by destrucktoid** [comment]
> still, majority of people arent going to know that xenobio fixes anything

Then that is their issue for not knowing. Nerfs should not be dealt out because people here lack the knowledge on how to fix it. Instead of nerfing things like this we should instead spread the knowledge of the ways to fix it. This trauma is in the game in it's current state for a reason and nerfing it in such a stupid way that makes it **trivially easy** to fix goes massively against the flavour and lore for heretics. It makes it so being sacrificed has basically no consequences since all medbay has to do is give you basic brain surgery and you are good again, literally a 1 minute job.

**Comment by destrucktoid** [comment]
> I would say it is good that the trauma can be easily fixed. But then we will see "willing sacrifices" happen more often as it has no consequences.

And that's kind of the problem. There SHOULD be consequences to being sacrificed, it SHOULD be more difficult than just "head to medbay get a 1 minute surgery".

**Comment by Majkl-J** [comment]
> I would say it is good that the trauma can be easily fixed. But then we will see "willing sacrifices" happen more often as it has no consequences.

We can add different consequences rather than just forcing a really annoying basically unfixable consequence whose fix relies on several people coordinating and xenobio actually being setup (Super unlikely, especially when you can barely talk or move around and dont have access to do it yourself).
I can guarantee at least half our playerbase does not have the knowledge on this or are not willing to go through the trouble and would rather cryo or get borged. I had people come to medbay or robo asking me to borg them because of the trauma making them unable to walk 5 feet without getting stunned/blinded.

**Comment by Majkl-J** [comment]
> > I would say it is good that the trauma can be easily fixed. But then we will see "willing sacrifices" happen more often as it has no consequences.
> 
> And that's kind of the problem. There SHOULD be consequences to being sacrificed, it SHOULD be more difficult than just "head to medbay get a 1 minute surgery".

Every other antag murking you is basically "head to medbay get a 1 minute surgery". It shouldn't be "Add annoying convoluted method to fix your specific problem". Medbay is there to.. surprise! fix most of your hurties, apart from body removal (Botany), and some antags like cult (Holy water).

**Comment by destrucktoid** [comment]
> > I would say it is good that the trauma can be easily fixed. But then we will see "willing sacrifices" happen more often as it has no consequences.
> 
> We can add different consequences rather than just forcing a really annoying basically unfixable consequence whose fix relies on several people coordinating and xenobio actually being setup (Super unlikely, especially when you can barely talk or move around and dont have access to do it yourself). I can guarantee at least half our playerbase does not have the knowledge on this or are not willing to go through the trouble and would rather cryo or get borged. I had people come to medbay or robo asking me to borg them because of the trauma making them unable to walk 5 feet without getting stunned/blinded.

You are aware that purple slimes, the ones needed for regen extracts are literally a t1 slime yes? You can make enough stuff for dozens of regen extracts in about 10-20 minutes from a completely unstarted xenobio. Also if people would rather cryo or get borged because they aren't willing to go about getting fixed, that is a PLAYER ISSUE, not a issue with the trauma. Also once more as i've said regarding your "player knowledge" argument. _Spread the knowledge_, if not enough people know about this then make more people aware of it; tell people, spread the knowledge and that will help make this "issue" less of one. The decision to nerf a trauma because of something that is, not even as a joke, a collective skill issue is bad.

**Comment by Majkl-J** [comment]
> > > I would say it is good that the trauma can be easily fixed. But then we will see "willing sacrifices" happen more often as it has no consequences.
> > 
> > 
> > We can add different consequences rather than just forcing a really annoying basically unfixable consequence whose fix relies on several people coordinating and xenobio actually being setup (Super unlikely, especially when you can barely talk or move around and dont have access to do it yourself). I can guarantee at least half our playerbase does not have the knowledge on this or are not willing to go through the trouble and would rather cryo or get borged. I had people come to medbay or robo asking me to borg them because of the trauma making them unable to walk 5 feet without getting stunned/blinded.
> 
> You are aware that purple slimes, the ones needed for regen extracts are literally a t1 slime yes? You can make enough stuff for dozens of regen extracts in about 10-20 minutes from a completely unstarted xenobio. Also if people would rather cryo or get borged because they aren't willing to go about getting fixed, that is a PLAYER ISSUE, not a issue with the trauma. Also once more as i've said regarding your "player knowledge" argument. _Spread the knowledge_, if not enough people know about this then make more people aware of it; tell people, spread the knowledge and that will help make this "issue" less of one. The decision to nerf a trauma because of something that is, not even as a joke, a collective skill issue is bad.

Medbay is the place you're meant to go to when you have troubles, not the asocial hermit that is the xenobiologist.
As for spreading the knowledge: This is literally the most convoluted game imaginable we should in no way expect each one of the players of various skill levels to know these things. I have seen people need help with basic surgery and you expect everyone to know "Ah yes I need to combine 10 core of this slime from xenobio with another adult one to fix my permanent trauma an antag gave me"
I would honestly rather just cryo/get borged than run around trying to get someone to actually make one the rest of my shift.

**Comment by destrucktoid** [comment]
> > > I would say it is good that the trauma can be easily fixed. But then we will see "willing sacrifices" happen more often as it has no consequences.
> > 
> > 
> > And that's kind of the problem. There SHOULD be consequences to being sacrificed, it SHOULD be more difficult than just "head to medbay get a 1 minute surgery".
> 
> Every other antag murking you is basically "head to medbay get a 1 minute surgery". It shouldn't be "Add annoying convoluted method to fix your specific problem". Medbay is there to.. surprise! fix most of your hurties, apart from body removal (Botany), and some antags like cult (Holy water).

When you get stabbed by a changeling or shot by a traitor the revival itself usually is where the issue lies. First you need to be found and then once you are found taken to medbay and depending on your wounds and how competent medbay is get revived in usually 5-10 minutes. Overall your average death takes anywhere from 10-20 minutes before revival depending upon method of death and medbay competence, also if you remembered to turn on sensors or not. Meanwhile heretics take you to funny rune, sacrifice you and you are back within 2 minutes with a trauma which is, again, nowhere near as debilitating as you seem to think for your average spessman. Then if you want to get the trauma fixed you can ask science to work on it and get it fixed almost instantly if someone is working xenobio, if not within about 15 minutes due to how easy purple slimes are to make. I do agree that i'd prefer if there was other consequences and/or another way to fix it, for example by visiting the psychologist or the chaplain and getting something from them (good avenues for RP there), but making it so that the trauma is easily fixable changes things so much in terms of consequences of death it's insane.

**Comment by Majkl-J** [comment]
> > > > I would say it is good that the trauma can be easily fixed. But then we will see "willing sacrifices" happen more often as it has no consequences.
> > > 
> > > 
> > > And that's kind of the problem. There SHOULD be consequences to being sacrificed, it SHOULD be more difficult than just "head to medbay get a 1 minute surgery".
> > 
> > 
> > Every other antag murking you is basically "head to medbay get a 1 minute surgery". It shouldn't be "Add annoying convoluted method to fix your specific problem". Medbay is there to.. surprise! fix most of your hurties, apart from body removal (Botany), and some antags like cult (Holy water).
> 
> When you get stabbed by a changeling or shot by a traitor the revival itself usually is where the issue lies. First you need to be found and then once you are found taken to medbay and depending on your wounds and how competent medbay is get revived in usually 5-10 minutes. Overall your average death takes anywhere from 10-20 minutes before revival depending upon method of death and medbay competence, also if you remembered to turn on sensors or not. Meanwhile heretics take you to funny rune, sacrifice you and you are back within 2 minutes with a trauma which is, again, nowhere near as debilitating as you seem to think for your average spessman. Then if you want to get the trauma fixed you can ask science to work on it and get it fixed almost instantly if someone is working xenobio, if not within about 15 minutes due to how easy purple slimes are to make. I do agree that i'd prefer if there was other consequences and/or another way to fix it, for example by visiting the psychologist or the chaplain and getting something from them (good avenues for RP there), but making it so that the trauma is easily fixable changes things so much in terms of consequences of death it's insane.

Back when it made you trade marbles for freedom you usually lost limbs so maybe that can be a tradeoff. Easy to do, just cut 2 random limbs, maybe some nonvital organ like tongue, eyes, or ears

**Comment by destrucktoid** [comment]
> > > > > I would say it is good that the trauma can be easily fixed. But then we will see "willing sacrifices" happen more often as it has no consequences.
> > > > 
> > > > 
> > > > And that's kind of the problem. There SHOULD be consequences to being sacrificed, it SHOULD be more difficult than just "head to medbay get a 1 minute surgery".
> > > 
> > > 
> > > Every other antag murking you is basically "head to medbay get a 1 minute surgery". It shouldn't be "Add annoying convoluted method to fix your specific problem". Medbay is there to.. surprise! fix most of your hurties, apart from body removal (Botany), and some antags like cult (Holy water).
> > 
> > 
> > When you get stabbed by a changeling or shot by a traitor the revival itself usually is where the issue lies. First you need to be found and then once you are found taken to medbay and depending on your wounds and how competent medbay is get revived in usually 5-10 minutes. Overall your average death takes anywhere from 10-20 minutes before revival depending upon method of death and medbay competence, also if you remembered to turn on sensors or not. Meanwhile heretics take you to funny rune, sacrifice you and you are back within 2 minutes with a trauma which is, again, nowhere near as debilitating as you seem to think for your average spessman. Then if you want to get the trauma fixed you can ask science to work on it and get it fixed almost instantly if someone is working xenobio, if not within about 15 minutes due to how easy purple slimes are to make. I do agree that i'd prefer if there was other consequences and/or another way to fix it, for example by visiting the psychologist or the chaplain and getting something from them (good avenues for RP there), but making it so that the trauma is easily fixable changes things so much in terms of consequences of death it's insane.
> 
> Back when it made you trade marbles for freedom you usually lost limbs so maybe that can be a tradeoff. Easy to do, just cut 2 random limbs, maybe some nonvital organ like tongue, eyes, or ears

The issue is that the idea is that you come back mentally scarred from the experience, you are meant to see horrors beyond mortal comprehension, not horrors so easily fixed that it's just another low level trauma to fix. I personally think that locking the trauma behind being fixed with a lobotomy is better in my opinion, that way it still keeps some of the consequences but if you are truly being affected by it you can still do something to fix it (at the risk of gaining another trauma instead). This also means the xenobio trick still works if you don't want to risk a lobotomy.

**Comment by Majkl-J** [comment]
> > > > > > I would say it is good that the trauma can be easily fixed. But then we will see "willing sacrifices" happen more often as it has no consequences.
> > > > > 
> > > > > 
> > > > > And that's kind of the problem. There SHOULD be consequences to being sacrificed, it SHOULD be more difficult than just "head to medbay get a 1 minute surgery".
> > > > 
> > > > 
> > > > Every other antag murking you is basically "head to medbay get a 1 minute surgery". It shouldn't be "Add annoying convoluted method to fix your specific problem". Medbay is there to.. surprise! fix most of your hurties, apart from body removal (Botany), and some antags like cult (Holy water).
> > > 
> > > 
> > > When you get stabbed by a changeling or shot by a traitor the revival itself usually is where the issue lies. First you need to be found and then once you are found taken to medbay and depending on your wounds and how competent medbay is get revived in usually 5-10 minutes. Overall your average death takes anywhere from 10-20 minutes before revival depending upon method of death and medbay competence, also if you remembered to turn on sensors or not. Meanwhile heretics take you to funny rune, sacrifice you and you are back within 2 minutes with a trauma which is, again, nowhere near as debilitating as you seem to think for your average spessman. Then if you want to get the trauma fixed you can ask science to work on it and get it fixed almost instantly if someone is working xenobio, if not within about 15 minutes due to how easy purple slimes are to make. I do agree that i'd prefer if there was other consequences and/or another way to fix it, for example by visiting the psychologist or the chaplain and getting something from them (good avenues for RP there), but making it so that the trauma is easily fixable changes things so much in terms of consequences of death it's insane.
> > 
> > 
> > Back when it made you trade marbles for freedom you usually lost limbs so maybe that can be a tradeoff. Easy to do, just cut 2 random limbs, maybe some nonvital organ like tongue, eyes, or ears
> 
> The issue is that the idea is that you come back mentally scarred from the experience, you are meant to see horrors beyond mortal comprehension, not horrors so easily fixed that it's just another low level trauma to fix. I personally think that locking the trauma behind being fixed with a lobotomy is better in my opinion, that way it still keeps some of the consequences but if you are truly being affected by it you can still do something to fix it (at the risk of gaining another trauma instead). This also means the xenobio trick still works if you don't want to risk a lobotomy.

I was gonna make it lobotomy if I got more feedback to make it such. I didnt

File: code/modules/antagonists/heretic/knowledge/sacrifice_knowledge/sacrifice_knowledge.dm [hunk 1]
```diff
@@ -377,10 +377,7 @@
 		var/datum/antagonist/heretic/victim_heretic = sac_target.mind?.has_antag_datum(/datum/antagonist/heretic)
 		victim_heretic.knowledge_points -= 3
 	else
-	//BUBBER EDIT START - makes the phobia curable
-		sac_target.gain_trauma(/datum/brain_trauma/mild/phobia/heresy, TRAUMA_RESILIENCE_SURGERY /*TRAUMA_RESILIENCE_MAGIC*/)
-	//BUBBER EDIT END
-
+		sac_target.gain_trauma(/datum/brain_trauma/mild/phobia/heresy, TRAUMA_RESILIENCE_MAGIC)
```
Comment (StrangeWeirdKitten) [inline code/modules/antagonists/heretic/knowledge/sacrifice_knowledge/sacrifice_knowledge.dm:380]:
```suggestion
                //Bubberstation Edit Original: TRAUMA_RESILIENCE_MAGIC
		sac_target.gain_trauma(/datum/brain_trauma/mild/phobia/heresy, TRAUMA_RESILIENCE_LOBOTOMY)
```

**Review by StrangeWeirdKitten — CHANGES_REQUESTED** [review]
I want it to be a lobotomy-level brain trauma.

**Comment by StrangeWeirdKitten** [comment]
Stale PR, but it's not going to get reverted. Reopen with a lobotomy level trauma if you still think it's too little.
